### DISCUSSION TITLE
Few questions for the developers

### DISCUSSION THREAD

**Discussion by robertmain53** [flyntwp/flynt#347] [Q&A] [answered]
A fantastic starter theme, congratulations. I have a few questions:

- how maintainable and reliable version 1.4.0 is expected to be in the long run? Do you expect components to be removed/deprecated? In other words, at this time how safe is use version 1.4.0 (base and premium) as a base to build themes for several " life-or-death" sites?
- Once I adopt Flynt I'm stuck to it unless I manually export content from ACFs to the_content(). This is true for all Timber-based themes, I guess. Do you have best practices to "move out of Flynt"?
- Do you have links to medium-size Magazine sites build with Flynt?

Cheers  on the great work you have done

**Comment by steffenbew** [comment, marked as answer]
Thanks @robertmain53! Happy to answer your questions:

- We're using Flynt v1.4.0 for [our own agency](https://bleech.de/en/) projects as well. As Flynt being a starter theme, we don't intend it to be updated for the sites that are built with it. New versions of Flynt are supposed to be used for new websites. That said, we're currently working on [Flynt v2.0](https://github.com/flyntwp/flynt/tree/next), which is going to focus on latest browser compatibility, to make use of CSS custom properties and to get rid of legacy code. If IE11 is supposed to be supported in a new project, Flynt v1.4 will still be a good choice, even after v2.0 is out.
- I think for custom built WordPress sites, you always need to take into consideration that most times, they're not supposed to be easily replaced by another theme. However, the only significant dependency of Flynt is ACF Pro. You could rebuilt a website from scratch, without Flynt, with just rendering the ACF data from the WordPress database.
- We've built the [Native Instruments Blog](https://blog.native-instruments.com/) and [finanzen.de](https://www.finanzen.de/) as 2 examples that might fall into that category.

Hope that helps!
